src/webgpu/api/operation/command_buffer/image_copy.spec.ts

```diff
@@ -625,6 +626,149 @@ class ImageCopyTest extends GPUTest {
         unreachable();
     }
   }
+
+  async CheckStencilTextureContent(
+    stencilTexture: GPUTexture,
+    stencilTextureSize: [number, number, number],
+    stencilTextureFormat: GPUTextureFormat,
+    expectedStencilTextureData: Uint8Array,
+    expectedStencilTextureDataOffset: number,
+    stencilTextureMipLevel: number
+  ): Promise<void> {
+    const copyLayout = getTextureCopyLayout('stencil8', '2d', stencilTextureSize, {
+      mipLevel: stencilTextureMipLevel,
+    });
+    assert(
+      expectedStencilTextureData.byteLength ===
+        copyLayout.mipSize[0] * copyLayout.mipSize[1] * copyLayout.mipSize[2] +
+          expectedStencilTextureDataOffset
+    );
+
+    const renderPipeline = this.device.createRenderPipeline({
+      vertex: {
+        module: this.device.createShaderModule({
+          code: `
+            [[stage(vertex)]] fn main(
```

Comment:
nit: the separation between the function declaration and its body is a bit hard to see. Ditto below for the fragment shader.

Reply:
Does it look better now?